### PR TITLE
Fs jul19 yevhen sag airbnb search bar

### PR DESCRIPTION
**Pull request by eugeneSahach**
- [DEMO LINK](https://yevhen-sahach.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://yevhen-sahach.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -13,7 +13,21 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-
+  <form action="#" class="search-bar-form" data-qa="big">
```
Comment (gestap) [inline src/index.html]: add attribute `method`. It's mandatory for forms

File: src/index.html [hunk 2]
```diff
@@ -13,7 +13,21 @@
...
+    <input
```
Comment (gestap) [inline src/index.html]: add attribute `name`. It's mandatory for inputs

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+  font-family: "Avenir";
+  src: url(./Fonts/Avenir/Avenir.ttf) format("truetype");
+  font-weight: normal;
+  font-style: normal;
+}
+
+.search-bar {
+  width: 100%;
+  height: 70px;
+  background: #fff;
```
Comment (gestap) [inline src/style.css]: I think this rule is no sense

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+}
+
+.search-bar::-webkit-input-placeholder {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.search-bar:-ms-input-placeholder {
```
Comment (gestap) [inline src/style.css]: remove these rules, you duplicate it.

use `::` for pseudo-elements

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+  height: 42px;
+  padding-left: 33px;
+}
+
+.search-bar--sm::-webkit-input-placeholder {
+  font-size: 14px;
+  line-height: 16px;
+}
+
+.search-bar--sm:-ms-input-placeholder {
```
Comment (gestap) [inline src/style.css]: remove these rules, you duplicate it.

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+}
+
+.search-bar-form--sm::after {
+  top: 15px;
+  left: 13px;
+  width: 11px;
+  height: 11px;
+  background-size: cover;
+}
+/*# sourceMappingURL=style.css.map */
```
Comment (gestap) [inline src/style.css]: document must end with one empty line
```suggestion
/*# sourceMappingURL=style.css.map */

```

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+  border-radius: 4px;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  padding-left: 62px;
+}
+
+.search-bar::-webkit-input-placeholder {
+  font-family: Avenir, sans-serif;
```
Comment (gestap) [inline src/style.css]: placeholder inherit it from `input`, no need to write it.

File: src/style.css [hunk 8]
```diff
@@ -0,0 +1,114 @@
...
+  width: 100%;
+  height: 70px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+  -webkit-box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+          box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: Avenir, sans-serif;
```
Comment (gestap) [inline src/style.css]: You can use `inherit` instead of duplicating this rule.

File: src/index.html [hunk 9]
```diff
@@ -13,7 +13,21 @@
...
+    type="text"
+    class="search-bar"
+    placeholder="Try &quot;Los Angeles&quot;"
+    data-qa="keypress"
+    >
+  </form>
+  <form action="#" class="search-bar-form search-bar-form--sm" data-qa="small">
```
Comment (gestap) [inline src/index.html]: Назви класів трохи не логічні. Наскільки я розумію ти брав за основу BEM.
Якщо так, тоді в тебе має бути блок, елементи та модифікатори.

Наприклад
`search-form search-form_small`
`search-form__input search-form__input_small`

File: src/style.css [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+}
+
+.search-form__input::-webkit-input-placeholder {
+  color: #3d4e61;
+}
+
+.search-form__input:-ms-input-placeholder {
+  color: #3d4e61;
+}
+
```
Comment (gestap) [inline src/style.css]: why you duplicate this class again?
`.search-form__input:-ms-input-placeholder` vs `.search-form__input::-ms-input-placeholder`

File: src/style.css [hunk 11]
```diff
@@ -0,0 +1,81 @@
...
+.search-form__input::-ms-input-placeholder {
+  color: #3d4e61;
+}
+
+.search-form__input::placeholder {
+  color: #3d4e61;
```
Comment (gestap) [inline src/style.css]: you duplicated this rule 4 times

File: src/index.html [hunk 12]
```diff
@@ -13,7 +13,23 @@
...
 <body>
-
+  <form method="GET" action="#" class="search-form" data-qa="big">
+    <input
+    type="text"
+    class="search-form__input"
+    name="search bar"
+    placeholder="Try &quot;Los Angeles&quot;"
+    data-qa="keypress"
+    >
```
Comment (gestap) [inline src/index.html]: Знімок екрана  о 3 51 06 пп
Знімок екрана  о 3 51 17 пп

Знімок екрана  о 3 56 52 пп

Comment (gestap) [reply src/index.html]: wrong place for the icon, it's too high.
When you typing, font-weight must change

File: src/style.css (hunk 11 above)
Comment (yevhen-sahach) [reply src/style.css]: I didn't do this, that's how compiler compiled my style.scss. I'll change it manually in .css file

File: src/index.html (hunk 12 above)
Comment (yevhen-sahach) [reply src/index.html]: On my machine it changes, maybe there is a problem with compatibility, but I don't know how to fix it. If you could help me with this I would be very thankfull!

File: src/index.html [hunk 13]
```diff
@@ -13,7 +13,23 @@
...
+  </form>
+  <form method="GET" action="#" class="search-form search-form_small" data-qa="small">
+    <input
+    type="text"
```
Comment (danheim) [inline src/index.html]: для переносимых аттрибутов отступ 2 пробела, они вложены для инпута

**Pull request closed without merging**